features/step_definitions/helper_services.rb

```diff
@@ -484,7 +484,7 @@
       res = host.exec_admin(*setup_commands)
       raise "iSCSI initiator setup commands error" unless res[:success]
     end
-    @result = admin.cli_exec(:create, n: _project.name, f: 'https://raw.githubusercontent.com/openshift-qe/docker-iscsi/master/iscsi-target.json')
+    @result = admin.cli_exec(:create, n: _project.name, f: "#{ENV['BUSHSLICER_HOME']}/testdata/storage/iscsi/iscsi-target.json")
```

Comment:
We can use `BushSlicer::HOME` to replace `ENV['BUSHSLICER_HOME']`

Reply:
Sure, let's follow the professional guidance :) 
Yes, I checked the other file, BushSlicer::HOME is more used, I updated the another one in this file.